conseil-api/src/main/resources/application.conf

```diff
@@ -19,7 +19,7 @@ conseil {
     keys: []
     keys+= ${?CONSEIL_API_KEY}
     
-    allow-blank: false
+    allow-blank: true
```

Comment:
by default it should be `false`, please be careful when you commit you configuration 😛